frontend/dockerfile/docs/reference.md

```diff
@@ -1125,15 +1125,15 @@ RUN apt-get update && apt-get install -y ...
 ADD has two forms:
 
 ```dockerfile
-ADD [--chown=<user>:<group>] [--checksum=<checksum>] <src>... <dest>
-ADD [--chown=<user>:<group>] ["<src>",... "<dest>"]
+ADD [--chown=<user>:<group>] [--chmod=<perms>] [--checksum=<checksum>] <src>... <dest>
+ADD [--chown=<user>:<group>] [--chmod=<perms>] ["<src>",... "<dest>"]
 ```
 
 The latter form is required for paths containing whitespace.
 
 > **Note**
```

Comment:
I think this should mention somewhere that it does not support octal notation, as mentioned here https://github.com/docker/cli/pull/3560#issuecomment-1102478924

Possibly with a link to https://github.com/moby/buildkit/issues/1951 if someone wants to track that?